### PR TITLE
to-render-text

### PR DESCRIPTION
**Pull request by albertfdp**
http://airbnb.io/enzyme/docs/api/ReactWrapper/text.html

```js
const wrapper = mount(<div><b>important</b></div>);
expect(wrapper, 'to render text', 'important');
```

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -27,6 +27,20 @@ const unexpectedEnzyme = {
...
+      (expect, reactWrapper) => {
+        return expect.shift(reactWrapper.text());
+      }
+    );
+
+    childExpect.exportAssertion(
+      '<ReactWrapper> [not] to render text <string>',
+      (expect, reactWrapper, text) => {
+        expect(reactWrapper.text(), '[not] to equal', text);
+      }
```
Comment (albertfdp) [inline src/index.js:54]: @sunesimonsen This is conflicting with the previous assertion, is there a way to handle that?

Comment (sunesimonsen) [reply src/index.js:54]: The other one should be called `to render text satisfying`. The to satisfy behavior can be pretty surprising, so we usually mark it explicit.

File: src/index.js [hunk 2]
```diff
@@ -27,6 +27,22 @@ const unexpectedEnzyme = {
       }
     );
 
+    childExpect.exportAssertion(
+      '<ReactWrapper> to render text satisfying <assertion>',
+      (expect, reactWrapper) => {
+        expect.errorMode = 'bubble';
```
Comment (sunesimonsen) [inline src/index.js]: Try `expect.errorMode = 'nested'` yields a better result.

**Review by sunesimonsen — APPROVED** [review]
👍 the assertion seems good, we can always tweak the error message if you want this merged now.

File: src/index.js (hunk 2 above)
Comment (albertfdp) [reply src/index.js]: Yeah, thanks

**Pull request merged**